contracts/staking-v1/src/state.rs

```diff
@@ -500,3 +500,42 @@ pub fn load_withdrawals(
         .may_load(storage, staker.as_slice())
         .map(|res| res.unwrap_or_default())
 }
+
+const MAX_LIMIT: u32 = 100;
+const DEFAULT_LIMIT: u32 = 10;
```

Comment:
can we put these const variables inside the function?
Isn't used anywhere else, and nobody knows for what this MAX_LIMIT is

Reply:
and can we increase MAX_LIMIT?

Or any reason that it's 100?

Reply:
to not to have a heavy json response message